dist/strap/include/checkout/tax_popup

```diff
@@ -1,4 +1,7 @@
-[if config SalesTax ne multi]
+[if type=!explicit compare="
+    local $_ = $Config->{SalesTax};
+    return $_ eq 'multi' || /^\[/;
```

Comment:
Is this now treating SalesTax set to an ITL tag similarly to `multi`? Just trying to understand this change.

Reply:
2nd condition expanded out to match on `/^\[tax[-_]lookup/` instead.